jest-puppeteer.config.js

```diff
@@ -8,7 +8,9 @@ module.exports = {
   },
   server: {
     launchTimeout: 60000,
-    command: 'PORT=8000 BROWSER=none yarn start',
+    // react-scripts start unconditionally sets the NODE_ENV to development,
+    // so we're setting CI here even if that isn't necessarily true
+    command: 'CI=true PORT=8000 BROWSER=none yarn start',
```

Comment:
oh, `NODE_ENV=production PORT=8000 BROWSER=none yarn start` does not work?

Reply:
i was trying to set it to `test` but cra overrides it on boot with `yarn start` so CI is just a way to tell you're in a testing mode so that it'll use the fixtures